test/Microsoft.ML.Functional.Tests/Training.cs

```diff
@@ -467,5 +467,37 @@ public void MetacomponentsFunctionAsExpectedOva()
             // Evaluate the model.
             var binaryClassificationMetrics = mlContext.MulticlassClassification.Evaluate(binaryClassificationPredictions);
         }
+
+        /// <summary>
+        /// Training: Meta-compononts function as expected. For OVA (one-versus-all), a user will be able to specify only
+        /// binary classifier trainers. If they specify a different model class there should be a compile error.
+        /// </summary>
+        [Fact]
+        public void MetacomponentsFunctionWithKeyHandeling()
```

Comment:
(nit) `Handeling`

Reply:
how it's different from what I have now?


---
In reply to: [271069790](https://github.com/dotnet/machinelearning/pull/3101#discussion_r271069790) [](ancestors = 271069790)

Reply:
did you mean `Handling`?

---
In reply to: [271072076](https://github.com/dotnet/machinelearning/pull/3101#discussion_r271072076) [](ancestors = 271072076,271069790)

Reply:
`Handeling` isn't a word (at least in English).